Brisk/AppDelegate.swift

```diff
@@ -68,7 +68,7 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
     }
 
     private func setupStatusItem() {
-        let image = NSImage(named: "StatusItemIcon")!
+        let image = NSImage(named: NSImage.Name(rawValue: "StatusItemIcon"))!
```

Comment:
Haha, super weird this isn’t the same as UIKit

Reply:
indeed! 😜

Reply:
could use an image literal, but I think it's not supported on older macs

Reply:
the docs say this type is 10.13+, but I assume that's not actually true